uaclient/entitlements/tests/test_esm.py

```diff
@@ -157,7 +157,7 @@ def fake_subp(cmd, capture=None, retry_sleeps=None):
                     entitlement.name
                 ),
                 "http://{}".format(entitlement.name.upper()),
-                "TOKEN",
+                "%s-token" % entitlement.name,
```

Comment:
A nit, but `.format` here would be more consistent with the rest of the codebase.